README.md

```diff
@@ -21,6 +21,27 @@ $ export KUBECONFIG=<directory_to_cloned_repo>/crc-tmp-install-data/auth/kubecon
 $ kubectl get pods --all-namespaces
 ```
 
+## Building SNC for OKD 4
+- Before running `./snc.sh`, you need to set some environment variables to override the default behavior.
+- Select the OKD 4 release that you want to build from: [https://origin-release.apps.ci.l2s4.p1.openshiftapps.com](https://origin-release.apps.ci.l2s4.p1.openshiftapps.com)
+- For example, to build release: 4.5.0-0.okd-2020-08-12-020541
+
+```bash
+export OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE="quay.io/openshift/okd:4.5.0-0.okd-2020-08-12-020541"
+export OPENSHIFT_VERSION=4.5.0-0.okd-2020-08-12-020541
+export MIRROR=https://github.com/openshift/okd/releases/download
+export OPENSHIFT_PULL_SECRET_PATH="/tmp/pull_secret.json"
+export BASE_OS=fedora-coreos
```

Comment:
Given that fedora-coreos is always the base os for OKD, we can really infer everything from OPENSHIFT_VERSION.  I'll submit another PR after this one with that logic, if it is approved.

I'll also simplify the guestfish logic as referenced below.

Reply:
If you only set `OPENSHIFT_VERSION`, you can't know if you are creating an OpenShift or OKD bundle, one way of handling this would be through separate variables for OpenShift (`OPENSHIFT_VERSION`) or OKD (`OKD_VERSION`), but there are many other ways of handling this (could be command line option, another env var, ...)

Reply:
I was planning to do it by parsing the VERSION string.  OKD always has something like: `4.5.0-0.okd-2020-09-04-180756` where OCP might just be 4.5.8 or a nightly string.  Using OPENSHIFT_VERSION vs. OKD_VERSION is more user friendly however.  So, we'll go that route.  Good call.

Reply:
relying on the format of the version string to guess OCP vs OKD feels error-prone, the format could change without any warnings.

Reply:
Agreed.  We'll go with the OPENSHIFT_VERSION vs. OKD_VERSION route.

Reply:
Modified to trigger OKD build based on OKD_VERSION